src/main/java/org/folio/service/organization/OrganizationStorageService.java

```diff
@@ -159,10 +174,12 @@ public CompletableFuture<Void> updateOrganizationById(String id, Organization up
 
   @Override
   public CompletableFuture<Void> deleteOrganizationById(String id, Context context, Map<String, String> headers) {
+    logger.debug("deleteOrganizationById:: Trying to delete organization by id: {}", id);
     HttpClientInterface client = getHttpClient(headers);
     return handleDeleteRequest(resourceByIdPath(ORGANIZATIONS, id), client, headers, logger).handle((org, t) -> {
       client.closeClient();
       if (Objects.nonNull(t)) {
+        logger.warn("Error deleting organization with id: {}", id, t);
         throw new CompletionException(t.getCause());
       }
       return null;
```

Comment:
is it okay to use logger.warn only inside of object.nonNull?
Or Should I use the above return null?

Reply:
Its okay to use only  inside of object.nonNull

Reply:
thanks